backend/rates/rates.go

```diff
@@ -122,6 +123,38 @@ func (updater *RateUpdater) Last() map[string]map[string]float64 {
 	return updater.last
 }
 
+// LastForPair returns the conversion rate for the given (coin, fiat) pair. Returns an error if the
+// rates have not been fetched yet.
+func (updater *RateUpdater) LastForPair(coinCode string, fiat string) (float64, error) {
+	// TODO: use coin.Code
+	last := updater.Last()
```

Comment:
I've been trying to avoid this because there's a concurrent use issue: `last` may be read and modified at the same time. Was saving it for later to add a mutex.